docs/uploading_and_publishing/explanation_how_the_processing_works.md

```diff
@@ -0,0 +1,122 @@
+
+The **mainfiles** are those files which are representative of a given computational calculation. The presence of a mainfile in the upload is key for NOMAD to recognize a calculation. In NOMAD, we support several computational codes for first principles calculations, molecular dynamics simulations, and lattice modeling, as well as workflow and database managers. For each code, NOMAD recognizes a single file as the mainfile. For example, the [VASP](https://www.vasp.at/) mainfile is by default the `vasprun.xml`, although if the `vasprun.xml` is not present in the upload NOMAD searches the `OUTCAR` file and assigns it as the mainfile[^1]. 
+
+
+[^1]: Please, check our note [References > VASP POTCAR stripping](refs.md/#vasp-potcar-stripping).
+
+
+The rest of files which are not the mainfile are recognized as **auxiliary files**. These can have several purposes and be supported and recognized by NOMAD. For example, the `band*.out` or `GW_band*` files in [FHI-aims](https://fhi-aims.org/) are auxiliary files that allows NOMAD to recognize band structures in DFT and GW, respectively.
+
+You can see the full list of supported codes, mainfiles, and auxiliary files in the general NOMAD documentation under [Supported parsers](https://nomad-lab.eu/prod/v1/staging/docs/reference/parsers.html).
+
+We recommend to keep as many auxiliary files as possible together with the mainfile, but without reaching the [uploads limits](refs.md/#uploads-limits). Please, also check our recommendations on [Best Practices: preparing the data and folder structure](refs.md/#best-practices-preparing-folder-upload).
+
+
+## Structured data with the NOMAD metainfo
+
+Once the mainfile has been recognized, a new [entry](../glossary/glossary.md/#entries) in NOMAD is created and a specific [parser](#parsing) is called. The auxliary files are searched by and accessed within the parser. You can check more details in [Writing a parser plugin](../writing_a_parser_plugin/parser_plugin_overview.md) on how to add new parsers in order for NOMAD to support new codes.
+
+For this new entry, NOMAD generates a **NOMAD archive** section. It will contain all the (meta)information extracted from the unstructured raw data files but in a _structured_, _well defined_, and _machine readable_ format. This **metadata** provides context to the raw data, i.e., what were the input methodological parameters, on which material the calculation was performed, etc. We define the **NOMAD Metainfo** as all the set of [sections, sub-sections](../glossary/glossary.md/#metainfo-sections), and [quantities](../glossary/glossary.md/#metainfo-quantities) used to structure the raw data into a structured _schema_. Further information about the NOMAD Metainfo is available in the general NOMAD documentation page in [Learn > Structured data](https://nomad-lab.eu/prod/v1/staging/docs/learn/data.html).
```

Comment:
nomad achive is not a section.

Reply:
I think we do not need to introduce nomad metainfo here as it is only the technology that enables us to create the metadata schema. We simply mention that the structure of the metadata is defined by a schema which can either be a nomad standard or generated by the user

Reply:
EntryArchive is a Section. It is the main section, in fact.

In any case, I would like to slightly touch on metainfo, in a simplistic way. I felt is sometimes hard to imagine these technologies without actually visualizing an easy and understandable case.

Reply:
> EntryArchive is a Section. It is the main section, in fact.
> 
> In any case, I would like to slightly touch on metainfo, in a simplistic way. I felt is sometimes hard to imagine these technologies without actually visualizing an easy and understandable case.

I would like to qualify my statement a bit, yes EntryArchive archive is a an msection by construction, but when we use nomad archive, for me it is more than simply the section itself, it comes along with it all the technologies built upon it. Anyway this is simple nomenclature.

Reply:
Ok, done.